_pages/how-we-work/sensitive-information.md

```diff
@@ -52,4 +55,4 @@ CircleCI has a [Build Pull Request from Forked repos setting](https://circleci.c
 GitHub has a [pull_request_target workflow trigger that can be misused by any repo owner](https://securitylab.github.com/research/github-actions-preventing-pwn-requests).
 
 #### Other references for protecting Sensitive Information in CI/CD
-* https://sprocketfox.io/xssfox/2021/01/18/pipeline/
+* [Build Pipeline Security](https://sprocketfox.io/xssfox/2021/01/18/pipeline/)
```

Comment:
A list of one probably shouldn't be a list. But I wouldn't call that a blocker :)